AbstractSQLRules2SQL.ometajs

```diff
@@ -743,6 +744,26 @@ export ometa AbstractSQLRules2SQL {
 			)
 		),
 
+	Case :indent =
+		[	'Case'
+			NestedIndent(indent):nestedIndent
+			[	When(indent)+:whens
```

Comment:
You probably want to pass `nestedIndent` here instead of `indent` to get nice formatting in the cases where `When` uses it